### PR TITLE
Noufalrahim/feat/added units to pressure sore

### LINKED ISSUE #8209: Pressure sore : width and height units are missing
**Is your feature request related to a problem? Please describe.**
[A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]](https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/edfe0bd6-0f50-4a4c-8cd0-2d4ad0e9cea1/daily_rounds/9ad27d33-0ced-4c11-9274-90cf8f86b316/update)

When marking push score (a scoring mechanism to understand how bad the sore is) for pressure sores , the width and high of the pressure sores are measured and logged in cms.

The form in CARE fails to mention the unit of measurement

**Describe the solution you'd like**
Screenshot 2024-07-24 at 3 26 42 PM
Add unit as cm

### PR DESCRIPTION
**Pull request by noufalrahim**
## Proposed Changes

- Fixes #8209 
- Added Units
- Added Units to i18n locale files (Consultation.json)
- More?

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Locale/en/Consultation.json [hunk 1]
```diff
@@ -76,5 +76,7 @@
   "diagnosis_at_discharge": "Diagnosis at Discharge",
   "procedure_suggestions": "Procedure Suggestions",
   "patient_notes_thread__Doctors": "Doctor's Discussions",
-  "patient_notes_thread__Nurses": "Nurse's Discussions"
+  "patient_notes_thread__Nurses": "Nurse's Discussions",
+  "width_label__cm": "Width (cm)",
+  "length_label__cm": "Length (cm)"
```
Comment (rithviknishad) [inline src/Locale/en/Consultation.json]:
```suggestion
  "width": "Width ({{ unit }})",
  "length": "Length ({{ unit }})"
```

File: src/Components/LogUpdate/Sections/PressureSore/PressureSore.tsx [hunk 2]
```diff
@@ -129,7 +132,7 @@ const RegionEditor = (props: RegionEditorProps) => {
         <br />
         <div className="grid grid-cols-2 gap-2">
           <TextFormField
-            label="Width"
+            label={t("width_label__cm")}
```
Comment (rithviknishad) [inline src/Components/LogUpdate/Sections/PressureSore/PressureSore.tsx]:
```suggestion
            label={t("width", { unit: "cm" })}
```

File: src/Components/LogUpdate/Sections/PressureSore/PressureSore.tsx [hunk 3]
```diff
@@ -141,7 +144,7 @@ const RegionEditor = (props: RegionEditorProps) => {
             onChange={(e) => update({ width: parseFloat(e.value) })}
           />
           <TextFormField
-            label="Length"
+            label={t("length_label__cm")}
```
Comment (rithviknishad) [inline src/Components/LogUpdate/Sections/PressureSore/PressureSore.tsx]:
```suggestion
            label={t("length", { unit: "cm" })}
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by bodhish** [comment]
@noufalrahim please add human firendly title and description for the PR. We also expect you to complete the #Merge Checklist

cc @rithviknishad

**Comment by nihal467** [comment]
@noufalrahim can you push the changes requested by @bodhish

**Pull request closed without merging**